py-polars/polars/functions/aggregation/vertical.py

```diff
@@ -1,14 +1,23 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from builtins import any as standard_any
+from typing import cast
 
 import polars.functions as F
+from polars.expr import Expr
 
-if TYPE_CHECKING:
-    from polars import Expr
 
+def into_expr(*names: str | Expr) -> Expr:
```

Comment:
You should use the util `parse_into_expr` to parse expression input.

Reply:
Thank you for the pointer! I did still have to do some parsing for the error check but I think it is ready now.